src/server/db/database.js

```diff
@@ -1,7 +1,11 @@
+// @flow
+
 import 'dotenv/config';
 import pgp from 'pg-promise';
 
 class Database {
+  db: any;
```

Comment:
More specific type?

Reply:
Short of completely writing the `pg-promise` type info, I'd have to put in the `any` type at some level. This was the fastest way to deal with it.